app/lib/activitypub/activity/create.rb

```diff
@@ -53,6 +53,7 @@ def process_status
     ApplicationRecord.transaction do
       @status = Status.create!(@params)
       attach_tags(@status)
+      attach_mentions(@status)
```

Comment:
Why do we need to call `attach_mentions` here? I am not sure I understand how it relates

Reply:
`attach_tags` was handling both hashtags and mentions, I split `attach_tags` into `attach_tags` and `attach_mentions` because I thought that was confusing

Reply:
Ha yes I missed the new `def attach_mentions` line in the diff, thanks!